pack/eoe/eoep.json

```diff
@@ -1033,221 +1670,766 @@
-        "traits": "Ally. Miskatonic.",
+        "skill_agility": 2,
+        "text": "Uses (2 resources). Replenish these resources at the start of each round.\nResources on Antiquary may be spent to pay for [[Favor]], [[Relic]], or [[Ritual]] cards.\n[fast] During a skill test on a [[Favor]], [[Relic]], or [[Ritual]] card, spend 1 resource from Antiquary: You get +1 skill value for this test.",
+        "traits": "Talent.",
         "type_code": "asset",
-        "xp": 2
+        "xp": 3
     },
     {
-        "code": "08109",
-        "cost": 2,
-        "deck_limit": 2,
-        "faction_code": "rogue",
-        "faction2_code": "mystic",
-        "illustrator": "Andreas Zafiratos",
-        "name": "Blur",
+        "code": "08125",
+        "cost": null,
+        "deck_limit": 1,
+        "faction_code": "neutral",
+        "illustrator": "Steve Ellis",
+        "name": "In the Thick of It",
         "pack_code": "eoep",
-        "position": 109,
-        "quantity": 2,
-        "skill_agility": 1,
-        "slot": "Arcane",
-        "text": "Uses (3 charges).\n[action] If Blur has charges remaining: <b>Evade.</b> For this evasion attempt, you may use [willpower] instead of [agility], and you get +1 skill value. If you succeed, spend 1 charge and you may take an additional action this turn. If you succeed by 0, take 1 damage.",
-        "traits": "Spell.",
+        "permanent": true,
+        "position": 125,
+        "quantity": 1,
+        "text": "Permanent. Limit 1 per deck. Purchase at deck creation\nWhen you purchase In the Thick of It, suffer 2 physical and/or mental trauma. Then, earn 3 experience.",
+        "traits": "Curse.",
         "type_code": "asset",
-        "xp": 1
+        "xp": 0
     },
     {
-        "code": "08111",
+        "code": "08126",
         "cost": 2,
         "deck_limit": 2,
-        "faction_code": "rogue",
-        "faction2_code": "mystic",
-        "illustrator": "Andreas Zafiratos",
-        "name": "Blur",
+        "faction_code": "neutral",
+        "illustrator": "Tiziano Baracchi",
+        "health": 2,
+        "name": "Heavy Furs",
         "pack_code": "eoep",
-        "position": 111,
+        "position": 126,
         "quantity": 2,
-        "skill_agility": 1,
-        "skill_willpower": 1,
-        "slot": "Arcane",
-        "text": "Uses (4 charges).\n[action] If Blur has charges remaining: <b>Evade.</b> You may use [willpower] instead of [agility] and get +2 skill value for this evasion attempt. If you succeed, spend 1 or 2 charges and you may take that many additional actions this turn. If you succeed by 0, take 2 damage.",
-        "traits": "Spell.",
+        "skill_combat": 1,
+        "slot": "Body",
+        "text": "[reaction] After you reveal a non-[auto_fail] symbol on a chaos token during a skill test you are perfoming, deal 1 damage to Heavy Furs: Cancel that chaos token and return it to the bag. Reveal a new chaos token.",
```

Comment:
Please can you check that the hyphen between non and [auto_fail] is definitely a hyphen and not an en dash? "non-[auto_fail]" not "non–[auto_fail]"

Reply:
I'm using Sublime text, I believe the hyphen is a normal hyphen. Will paste one just in case?